nss-day-cohort-48/fun-website-1-faith-s-children
# My beginning code on East Hikes

Beginning east tn website.  Added page header, hike section name, hike images, hike specs.

## Steps to test

1. Pull branch
2. open chrome
3. Verify I have  page header, hike section name, hike images, hike specs
